public/app/plugins/datasource/grafana-testdata-datasource/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@grafana-plugins/grafana-testdata-datasource",
+  "description": "Generates test data in different forms",
+  "private": true,
+  "version": "10.2.0-pre",
+  "dependencies": {
+    "@emotion/css": "11.11.2",
+    "@grafana/data": "10.2.0-pre",
+    "@grafana/experimental": "1.7.0",
+    "@grafana/runtime": "10.2.0-pre",
+    "@grafana/schema": "10.2.0-pre",
+    "@grafana/ui": "10.2.0-pre",
+    "react": "18.2.0",
+    "react-use": "17.4.0",
+    "rxjs": "7.8.1",
+    "tslib": "2.6.0"
+  },
+  "devDependencies": {
+    "@grafana/e2e-selectors": "10.2.0-pre",
+    "@grafana/plugin-configs": "10.2.0-pre",
+    "@testing-library/react": "14.0.0",
+    "@testing-library/user-event": "14.4.3",
+    "@types/jest": "29.5.4",
+    "@types/lodash": "4.14.195",
+    "@types/node": "18.16.16",
+    "@types/react": "18.2.15",
+    "@types/testing-library__jest-dom": "5.14.8",
+    "lodash": "4.17.21",
+    "swc-loader": "0.2.3",
+    "ts-node": "10.9.1",
+    "webpack": "5.88.1"
+  },
+  "peerDependencies": {
+    "@grafana/runtime": "*"
+  },
+  "scripts": {
+    "build": "webpack -c ./webpack.config.ts --env production",
+    "build:commit": "webpack -c ./webpack.config.ts --env production --env commit=$(git rev-parse --short HEAD)",
+    "dev": "webpack -w -c ./webpack.config.ts --env development"
```

Comment:
Are there plans for a follow up PR which adds tests and linting scripts? I imagine the CI for publishing a plugin separately would probably run the plugin tests and lint only the plugins code rather than the whole jungle?

Reply:
I am not sure about that. For publishing the plugin, it should reach `main` so the tests and linter should already be passed. Do you think re-running those could catch some other errors?